rasa/core/channels/twilio.py

```diff
@@ -10,7 +10,7 @@
 from rasa.core.channels.channel import UserMessage, OutputChannel
 
 if TYPE_CHECKING:
-    from twilio.rest.api.v2010.account.message import MessageInstance
+    from twilio.rest.api.v2010.account.message import MessageInstance  # noqa: F401
```

Comment:
sounds a bit weird to require this comment, can't vulture respect `TYPE_CHECKING`?

Reply:
yeah - it sometimes works and sometimes doesn't (e.g. it doesn't raise for any other imports in `TYPE_CHECKING`)

Reply:
sounds a potential for flakiness. What can we do about it? Is this an issue Vulture people know about?

Reply:
Funny enough: They use `rasa_nlu` to discuss the handling of `if TYPE_CHECKING` 😆

Reply:
omg, our flake8 config is evil! Will probably dump vulture and use flake8 instead

Reply:
wdym?

Reply:
we configured flake8 to not flag unused imports 🙈